DuckDuckGo/TabViewController.swift

```diff
@@ -874,13 +876,13 @@ extension TabViewController: WKNavigationDelegate {
         }
         
         guard let documentUrl = navigationAction.request.mainDocumentURL else {
-            completion(.allow)
+            completion(allowPolicy)
             return
         }
         
         if shouldReissueSearch(for: url) {
             reissueSearchWithStatsParams(for: url)
-            completion(.cancel)
+            completion(allowPolicy)
```

Comment:
@brindy This change made ATB test fail, as we now don't reissue search as we used to.

Reply:
Thanks @bwaresiak - the system is working!! :D